styles.css

```diff
@@ -0,0 +1,181 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  body {
+    display: flex;
+    flex-direction: column;
+    font-family: "Roboto", Arial, sans-serif;
+  }
+
+  a:hover, button:hover {
+    cursor: pointer;
+   }
+
+  .hero, .footer {
+    background-color: #1F2937;
+  }
+
+  .container1 {
+    max-width: 900px;
+  }
+
+  .container2 {
+    max-width: 700px;
+  }
+
+  .container1, .container2 {
+    margin: auto;
+    padding-bottom: 100px
+  }
+
+  .top, .bottom {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .top {
+    margin-bottom: 100px;
+    padding: 8px 0;
+  }
+
+  .hero h1, button, .text {
+    color: #F9FAF8;
+  }
+
+  .top h1 {
+    font-size: 24px;
+  } 
+  
+  .bottom h1 {
+    font-size: 48px;
+    font-weight: 900;
+  }
+
+  .hero ul {
+    list-style-type: none;
+    display: flex;
+  }
+
+  .hero a {
+    text-decoration: none;
+    color: #E5E7EB;
+    font-size: 18px;
+  }
+
+  .bottom p {
+    color: #E5E7EB;
+    font-size: 18px;
+    padding-bottom: 10px;
+  }
+
+  .hero li {
+    padding: 0 8px;
+  }
+
+  .hero-image {
+    padding: 100px 180px;
+    margin-left: 30px;
+    background-color: grey;
+    color:#E5E7EB
+  }
+
+  button {
+    background-color: #3882f6;
+    padding: 6px 24px;
+    border-radius: 5px;
+    border: 0;
+  }
+
+.container2 h1 {
+    color: #1F2937;
+}
+
+.section-title {
+    text-align: center;
+    margin: 40px 0;
+    font-size: 36px;
+    font-weight: 900;
+}
+
+ .images {
+    display: flex;
+    justify-content: space-between;
+    gap: 32px;
+ }
+
+ .blank-img {
+    padding: 70px;
+    border: 2px solid #3882f6;
+    border-radius: 8px;
+    width: 0;
+    margin-bottom: 6px;
+ }
+
+ figcaption {
+    text-align: center;
+ }
+
+.second {
+    background-color: #E5E7EB;
+}
+
+.second p {
+    font-size: 36px;
+    font-style: italic;
+    font-weight: 300;
+    padding-left: 16px;
+    margin-top: 100px;
+}
+
+.second h1 {
+    text-align: end;
+    font-size: 24px;
+    font-weight: 700;
+    padding-top: 16px;
+    margin-right: 50px;
+}
+
+#cta {
```

Comment:
Typically ID's on attributes aren't meant to be used for styling. I would add a second classname on that tag to be cta or something instead.

Normally IDs are used for doing a `getElementById` in JS or making an anchor link work within a page